src/event.js

```diff
@@ -3,12 +3,12 @@ define( [
 	"./var/document",
 	"./var/documentElement",
 	"./var/rnothtmlwhite",
+	"./var/rcheckableType",
 	"./var/slice",
 	"./data/var/dataPriv",
-
 	"./core/init",
 	"./selector"
-], function( jQuery, document, documentElement, rnothtmlwhite, slice, dataPriv ) {
+], function( jQuery, document, documentElement, rnothtmlwhite, slice, dataPriv, rcheckableType ) {
```

Comment:
The parameter names need to line up with `define`d dependencies, so `rcheckableType` should precede `slice`.